ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -640,6 +664,21 @@ func (r *RayJobReconciler) getOrCreateRayClusterInstance(ctx context.Context, ra
 	return rayClusterInstance, nil
 }
 
+func (r *RayJobReconciler) getRayClusterInstance(ctx context.Context, rayJobInstance *rayv1.RayJob) (*rayv1.RayCluster, error) {
```

Comment:
I am not sure if we need this function. Perhaps we could directly use `RayJobRayClusterNamespacedName` and `r.Get(...)`?

Reply:
Great catch, will update to just call RayJobRayClusterNamespacedName